src/lib/bootloader/device_map.rb

```diff
@@ -92,9 +94,7 @@ def reduce_to_bios_limit
         return false
       end
 
-      grub_devices = @model.grub_devices
-      grub_devices.select! { |d| d.start_with?("hd") }
-      grub_devices.sort_by(&:to_i)
```

Comment:
@jreidinger @teclator This is actually a bug fix, because
```rb
(irb)> %w(hd3 hd0 hd1 hd2).sort_by(&:to_i)
=> ["hd3", "hd0", "hd1", "hd2"]
```

But since this is the cfa branch, nobody has probably seen the bug anyway.